build.xml

```diff
@@ -276,7 +276,13 @@
     <condition property="os.prefix" value="win32-${jre.arch}">
       <equals arg1="${build.os.family}" arg2="windows"/>
     </condition>
-    <!-- Darwin builds are universal, no arch required -->
+    <condition property="os.prefix" value="darwin-aarch64">
```

Comment:
A real machine calls arch as `arm64`

Reply:
So does Windows, but this project uses `aarch64`.  See also #1264.